source/HashTable.cpp

```diff
@@ -1,10 +1,9 @@
-#include "stdafx.h"
 #include "HashTable.h"
 
-
+//default number of groups in table
 const int HashTable::startSize = 16;
 
-
+//constructor without arguments
 HashTable::HashTable()
 {
```

Comment:
Please, move the brackets to the line with the function declaration for all functions